### PR TITLE
Update tests and add javadocs for Application, Internship, util, wrapper classes

### PR DESCRIPTION
**Pull request by keanecjy**
- Improve code quality
- Create util classes in commons.util package

### CODE REVIEW

File: src/main/java/seedu/address/model/internship/InternshipItem.java [hunk 1]
```diff
@@ -34,7 +40,13 @@
     private Set<Requirement> requirements = new HashSet<>();
 
     /**
-     * Every field must be present and not null.
+     * Constructs an InternshipItem object. Every field must be present and not null.
```
Comment (seanjyjy) [inline src/main/java/seedu/address/model/internship/InternshipItem.java:43]: I realized for constructors, in our main code, we use Creates, so maybe we could stick to the same javadocs for cohesiveness

File: src/main/java/seedu/address/model/util/InternshipItemUtil.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package seedu.address.model.util;
+
+/**
+ * InternshipItemUtil class which provides the strings used in the InternshipItem.
```
Comment (seanjyjy) [inline src/main/java/seedu/address/model/util/InternshipItemUtil.java]: Maybe can just remove the InternshipItemUtil class which and just be Provides the string used in ... ? or perhaps A utility class that provides ...

File: src/main/java/seedu/address/model/util/InternshipItemUtil.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+ */
+public class InternshipItemUtil {
+
+    public static final String COMPANY_OUTPUT_NAME = " Company Name: ";
+    public static final String PERIOD_OUTPUT_NAME = " Company Name: ";
+    public static final String WAGE_OUTPUT_NAME = " Wage: ";
```
Comment (seanjyjy) [inline src/main/java/seedu/address/model/util/InternshipItemUtil.java:10]: Sorry may I know what's the difference between `COMPANY_OUTPUT_NAME` and `PERIOD_OUTPUT_NAME`
could they just be summarized into`COMPANY_NAME = "Company Name: "`

**Review by seanjyjy — COMMENTED** [review]
LGTM, just some nits to fix

File: src/main/java/seedu/address/model/util/InternshipItemUtil.java (hunk 3 above)
Comment (keanecjy) [reply src/main/java/seedu/address/model/util/InternshipItemUtil.java:10]: Good catch, missed this completely

File: src/main/java/seedu/address/model/internship/InternshipItem.java (hunk 1 above)
Comment (keanecjy) [reply src/main/java/seedu/address/model/internship/InternshipItem.java:43]: I think it really depends on what context is it being used in. There seems to be an almost equal amount of Creates and Constructs used in our code atm.

File: src/main/java/seedu/address/model/util/InternshipItemUtil.java (hunk 2 above)
Comment (keanecjy) [reply src/main/java/seedu/address/model/util/InternshipItemUtil.java]: Yup, fixed in my latest merge.

**Comment by seanjyjy** [comment]
LGTM

**Pull request merged**